share/eds/store.go

```diff
@@ -36,6 +37,8 @@ const (
 	defaultGCInterval = time.Hour
 )
 
+var ErrItemNotExist = errors.New("item does not exist in store")
```

Comment:
```suggestion
var ErrItemNotExist = errors.New("eds: item does not exist in store")
```

Reply:
fine w me, let's just say 
`eds: not found in store`